src/services/earn/earn-service.ts

```diff
@@ -962,13 +937,13 @@ export class EarnService implements IEarnService {
 
     const arbitraryCall = this.permit2Service.arbitrary.buildArbitraryCallWithoutPermit({
       allowanceTargets: Object.entries(allowanceTargets).map(([token, target]) => ({ token, target })),
-      calls,
+      calls: [],
```

Comment:
Now that there are no `calls`, and if we send the tokens directly to the recipient in `sellOrderSwap`, then it could happen that `distributions` is empty. So we should check that before building `arbitraryCall` and adding it to the multicalls